prisma/schema.prisma

```diff
@@ -76,28 +76,54 @@ model Org {
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
   users     Member[]
+  roles     Role[]
 }
 
-enum AccessEnum {
-  READ
-  WRITE
-  ADMIN
+enum MEMBERHIP_STATUS {
+  ACCEPTED
+  PENDING
+  DECLINED
 }
 
 model Member {
-  id        String   @id @default(cuid())
-  title     String?
-  access    AccessEnum @default(READ)
-  active    Boolean  @default(true)
+  id           String           @id @default(cuid())
+  title        String?
+  active       Boolean          @default(true)
+  status       MEMBERHIP_STATUS @default(PENDING)
```

Comment:
added fields `status`, `invitedName`, `invitedEmail` and  `role`
added `invitedEmail` to unique array

removed field `access`

Reply:
We def need this, amazing 😍

Reply:
Turns out we dont need `invitedName`, `invitedEmail`, we will check if user exist with given email address, if they do, just create an invite link using next-auth and send an email. If the user does not exist, we will create a user record, send an invite email. We are keeping track of `MEMBERSHIP_STATUS` on membership table anyway, so we are good here.